types/react-native/test/globals.tsx

```diff
@@ -0,0 +1,3 @@
+const fetchCopy: GlobalFetch['fetch'] = fetch;
+
+const requestInfoAsString: RequestInfo = 'request';
```

Comment:
Please use real world code examples, e.g. taken from [here](https://github.com/mdn/fetch-examples) or possibly React Native examples, and preferably without explicit typing where possible.